tests/render/test_microsoft_teams_renderer.py

```diff
@@ -1,167 +1,43 @@
-from unittest.mock import Mock  # noqa: TID251
-
 import pytest
 
-from great_expectations.core import (
-    ExpectationSuiteValidationResult,
-    RunIdentifier,
-)
-from great_expectations.core.batch import LegacyBatchDefinition
-from great_expectations.data_context.types.resource_identifiers import (
-    BatchIdentifier,
-    ExpectationSuiteIdentifier,
-    ValidationResultIdentifier,
-)
+from great_expectations.checkpoint.checkpoint import CheckpointResult
 from great_expectations.render.renderer import MicrosoftTeamsRenderer
 
-# module level markers
-pytestmark = pytest.mark.big
 
+@pytest.mark.unit
+def test_MicrosoftTeamsRenderer_render(v1_checkpoint_result: CheckpointResult):
```

Comment:
Was missing this so just backfilling

Reply:
Can we rename the `v1_checkpoint_result` fixture to `checkpoint_result`?

Reply:
we unfortunately already have a fixture with that name - I haven't yet deleted the legacy checkpoint config